Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -964,6 +964,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEAD_CODE_STRIPPING = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"Basic-Car-Maintenance/Preview Content\"";
+				DEVELOPMENT_TEAM = "";
```

Comment:
you've selected development team and set it to none, but the correct way to fix this, is to manually go into the pbxproj file in like VSCode and delete the line that gets created (because without it, this will set everyone's team to none, rather than using the xcconfig file like it should)